### DISCUSSION TITLE
Will Modern.Forms be able to run GitExtensions?

### DISCUSSION THREAD

**Discussion by rgl** [modern-forms/Modern.Forms#39] [General]
I'm curious about your goals with Modern.Forms.

Would running a mature application like GitExtensions be something you have in mind?

BTW, https://github.com/gitextensions/gitextensions/releases/tag/v4-alpha1 is the first version running with dotnet 6.

**Comment by jpobst** [comment]
It's probably important to be realistic about what one person can accomplish writing an entire GUI toolkit as a side project.  Being able to run a complex application is likely a long way off, if ever.

**Comment by Ultrafeel** [comment]
> I'm curious about your goals with Modern.Forms.
> 
> Would running a mature application like GitExtensions be something you have in mind?
> 
> BTW, https://github.com/gitextensions/gitextensions/releases/tag/v4-alpha1 is the first version running with dotnet 6.

I think  the [DanielVanNoord/System.Windows.Forms](https://github.com/DanielVanNoord/System.Windows.Forms) , which ports Mono's WinForms to .NET 6+ to enable cross-platform support , fits more for GitExtensions , considering that it was using Mono till 2.51 version , which the last that supports Gnu/Linux.
